docs/documentation/cn/sql-reference/sql-functions/aggregate-functions/stddev.md

```diff
@@ -0,0 +1,19 @@
+# STDDEV
```

Comment:
STDDEV is same as STDDEV_SAMP，so I think you can write them toghter, like: STDDEV/STDDEV_SAMP